src/haz3lcore/pretty/ExpToSegment.re

```diff
@@ -54,7 +55,6 @@ let rec external_precedence = (exp: Exp.t): Precedence.t => {
 
   // Other forms
   | UnOp(Meta(Unquote), _) => Precedence.unquote
-  | Constructor(_) // Constructor is here because we currently always add a type annotation to constructors
```

Comment:
@Negabinary this comment seems outdated as far as I can tell we don't add the type annotations onto the constructors